src/test/java/io/appium/java_client/android/UIAutomator2Test.java

```diff
@@ -5,106 +5,84 @@
 
 import io.appium.java_client.MobileBy;
 import io.appium.java_client.MobileElement;
-import io.appium.java_client.remote.AutomationName;
-import io.appium.java_client.remote.MobileCapabilityType;
-import io.appium.java_client.service.local.AppiumDriverLocalService;
-import io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException;
 import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.DeviceRotation;
-import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-import java.io.File;
-import java.util.concurrent.TimeUnit;
 
-public class UIAutomator2Test {
-    private static AppiumDriverLocalService service;
-    protected static AndroidDriver<AndroidElement> driver;
+public class UIAutomator2Test extends BaseAndroidTest {
```

Comment:
this class can be deleted and all the tests can be moved to `AndroidDriverTest`.

Reply:
`AndroidDriverTest` is not running in CI currently. Can this be done later?

Reply:
@SrinivasanTarget ping

Reply:
Ok make travis happy